ReplicationAcceptance/ReplicationAcceptance/ReplicationAcceptance.m

```diff
@@ -128,7 +148,28 @@ -(CDTReplicator *) pullFromRemoteWithFilter:(NSString*)filterName params:(NSDict
         [NSThread sleepForTimeInterval:1.0f];
         NSLog(@" -> %@", [CDTReplicator stringForReplicatorState:replicator.state]);
     }
-    
+
+    XCTAssertGreaterThanOrEqual(interceptor.timesRequestIntercepted, 1);
+    int timesRequestIntercepted = interceptor.timesRequestIntercepted;
+
+    [pull clearInterceptors];
+
+    replicator = [self.replicatorFactory oneWay:pull error:&error];
+    XCTAssertNil(error, @"%@", error);
+    XCTAssertNotNil(replicator, @"CDTReplicator is nil");
+
+    NSLog(@"Replicating from %@", [pull.source absoluteString]);
+    if (![replicator startWithError:&error]) {
+        XCTFail(@"CDTReplicator -startWithError: %@", error);
+    }
+
+    while (replicator.isActive) {
+        [NSThread sleepForTimeInterval:1.0f];
+        NSLog(@" -> %@", [CDTReplicator stringForReplicatorState:replicator.state]);
+    }
+
+    XCTAssertEqual(timesRequestIntercepted, interceptor.timesRequestIntercepted);
+
```

Comment:
It adds a check to make sure the interceptors get passed down, it was a path of least resistance, going to revert, and we can work on it in a different PR, I already have a test that makes sure the interceptors are passed down to the change tracker